udp_driver/src/udp_socket.cpp

```diff
@@ -160,6 +160,11 @@ bool UdpSocket::isOpen() const
   return m_udp_socket.is_open();
 }
 
+void UdpSocket::reuseAddress()
+{
+  m_udp_socket.set_option(udp::socket::reuse_address(true));
```

Comment:
It seems unnecessary to add a new function for this since it's only used when `open()` is called and it is unlikely that someone would call this function by itself. Unless I'm missing something, please just move this line into the `open()` function.

Reply:
@JWhitleyWork fixed at https://github.com/ros-drivers/transport_drivers/pull/64/commits/211fb946ed0c5422778f914a1c7c65b3e434c6bb